io/zenoh-transport/src/common/pipeline.rs

```diff
@@ -863,8 +880,10 @@ impl TransmissionPipelineConsumer {
     }
 
     pub(crate) fn refill(&mut self, batch: WBatch, priority: Priority) {
-        self.stage_out[priority as usize].refill(batch);
-        self.status.set_congested(priority, false);
+        if !batch.is_ephemeral() {
+            self.stage_out[priority as usize].refill(batch);
+            self.status.set_congested(priority, false);
```

Comment:
@Mallets The merge conflict was on these lines. Just check I took the right decision about calling `set_congested` only if the batch is not ephemeral.

Reply:
That's the right decision.